alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
If you like the explicit if better lets leave it.

About the empty list, try this:
```python
def f(x=[]):
    x.append(1)
    return x
print(f())
print(f())
```
It is generally advised that one never uses mutable types for default params.